quic/s2n-quic-transport/src/path/manager.rs

```diff
@@ -418,8 +418,15 @@ impl<Config: endpoint::Config> Manager<Config> {
             .active_path()
             .rtt_estimator
             .for_new_path(limits.initial_round_trip_time());
-        let path_info =
-            congestion_controller::PathInfo::new(mtu_config.initial_mtu, &remote_address);
+
+        let info = mtu::PathInfo::new(&remote_address);
```

Comment:
nit: you could just construct the `PathInfo` inside the `mtu::Manager` since `mtu::Manager` isn't a public type, so we can change the `config` function as necessary. Not required though